src/nanotron/models/base.py

```diff
@@ -58,7 +58,7 @@ def params_gen():
         yield from params_gen()
 
     @abstractmethod
-    def init_model_randomly(self, init_method, scaled_init_method):
+    def init_model_randomly(self, config):
```

Comment:
this could break other parts of nanotron like doremi https://github.com/huggingface/nanotron/blob/9f117eeb1e8309c45e2b1820069292ed5d7c4647/examples/doremi/doremi/llama.py#L29